packages/contracts/src/dollar/utils/CollectableDust.sol

```diff
@@ -33,24 +33,30 @@ abstract contract CollectableDust is ICollectableDust {
             _protocolTokens.contains(_token),
             "collectable-dust::token-not-part-of-the-protocol"
         );
-        _protocolTokens.remove(_token);
+        require(_protocolTokens.remove(_token));
         emit ProtocolTokenRemoved(_token);
     }
 
-    function _sendDust(address _to, address _token, uint256 _amount) internal {
+    function _sendDust(
+        address to,
+        address token,
+        uint256 amount
+    ) internal {
         require(
-            _to != address(0),
+            to != address(0),
             "collectable-dust::cant-send-dust-to-zero-address"
         );
         require(
-            !_protocolTokens.contains(_token),
+            !_protocolTokens.contains(token),
             "collectable-dust::token-is-part-of-the-protocol"
         );
-        if (_token == ETH_ADDRESS) {
-            payable(_to).transfer(_amount);
+        if (token == ETH_ADDRESS) {
+            // slither-disable-next-line low-level-calls
+            (bool sent, ) = payable(to).call{value: amount}("");
+            require(sent, "Failed to transfer Ether");
```

Comment:
I changed this from
`payable(_to).transfer(_amount);`
to 
```(bool sent, ) = payable(to).call{value: amount}("");
require(sent, "Failed to transfer Ether");```
as this is the current recommended pattern for sending Ether.

This lead to a low-level-call warning from Slither so I put an ignore in. This is the only place I had slither ignore something

Reply:
Thanks for the heads up. we can ignore it from slither config.

Reply:
If from the Slither config, will it ignore all low level calls? Or can you specify for just this one? If former I wouldn't recommend it.

Reply:
It will ignore all low level calls right

Reply:
We shouldn't do that then. It's my recommendation that this specific instance gets a pass, but we would still want to investigate other calls.